apache-mesos-mixin/README.md

```diff
@@ -0,0 +1,113 @@
+# Apache Mesos Mixin
```

Comment:
Generally, we use Sentence case in Grafana:
https://grafana.com/docs/writers-toolkit/writing-guide/markdown-guide/#markdown-guide

Reply:
I went through and updated a couple areas. Let me know if there is anything that is not satisfactory. There are a couple places that end in a `:` followed by a list, but that format has been used for a lot of dashboards. Maybe that should be changed though?